tests/e2e/specs/shopper/checkout-redirect.test.js

```diff
@@ -2,3 +2,27 @@
  * Internal dependencies
  */
 import { shopper } from '../../../utils';
+import { BASE_URL } from '../../utils';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
```

Comment:
`block.name` seems undefined here. Given that this part only gets called for code that is not hidden behind a flag, I'd change that part from `skipping ${ block.name } tests` to `Skipping checkout tests` or to `Skipping redirection tests`.

In general, I'd question this part, as I believe that we should, at least, also test functionality that behind the featured flag. But that's a different story.